content/english/sql_workshop/activity1/_activity1.md

```diff
@@ -33,7 +34,7 @@ You are given information about the Galaxy and want to look for the Planet of Fu
 #### Question: What command will display all the columns in the database called "galaxy"?
 
 {{< rawhtml >}}
-<div class="terminal_div" id="terminal_div"><img class="terminal" src="../assets/Terminal.png" alt="Terminal_screen">
+<div class="terminal_div" id="terminal_div"><img class="terminal" src="../../media/Terminal.png" alt="Terminal_screen">
```

Comment:
What's the difference between media and assets?

Reply:
Media is the global file for the images we are using so all the activities can use them. Assets is the local file to store the images.